### DISCUSSION TITLE
missing features [Oct 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4266] [Discord Threads]
> 🙋‍♂️ **ryzenway**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### Previously we rewrote the frontend. May I know what are missing?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1032191903405449226)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
